scripts/deploy-rpc-playbooks.sh

```diff
@@ -36,13 +36,15 @@ if [[ "${DEPLOY_ELK}" == "yes" ]]; then
   run_ansible setup-logging.yml
 fi
 
+# Download the latest release of rpc-maas
+run_ansible maas-get.yml
```

Comment:
this was done because the rpc-gating system disables sets DEPLOY_MAAS to galse even though it deploys maas.